README.md

```diff
@@ -9,16 +9,19 @@ This code base is meant to categorize each line of a *python* script several dif
 * function defs
 * assignments
 * function calls
+* indentation levels / scope levels?
+
+# Additional functionality
 * hash lines and storage
 * command line calling
-
+* finding matching function calls and defs
+*
 
 
 # Todo List
 --------------------------- After this then I start making the tool that will be used for establishing relationships for a single script.
 * loops
```

Comment:
Either need to do this next time or varriable tracing... which might have to do with line ast trees from the python library.